modules/RTC/RTC.js

```diff
@@ -40,11 +41,20 @@ export default class RTC extends Listenable {
     constructor(conference, options = {}) {
         super();
         this.conference = conference;
+        /**
+         * A map of active <tt>TraceablePeerConnection</tt>.
+         * @type {Object.<number, TraceablePeerConnection>}
+         */
+        this.peerConnections = { };
+        /**
+         * The counter used to generated id numbers assigned to peer connections
+         * @type {number}
+         */
+        this.peerConnIdCounter = 1;
```

Comment:
if the previous var is called `peerConnections` this one should follow and use the complete name IMHO